CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ The releases of major and minor versions contain an overview of changes since th
 Version 1.8.x
 -------------
 
+## Version 1.8.2
+- Print all linked libraries when using `--version`.
+- Developer: Require at least CMake version 3.15.
+- Developer: Use Dockerfile in CI.
```

Comment:
For developers, it may be good to highlight you moved the storm-config.h.in ;-)

Reply:
Yes, good idea.